data/os/Ubuntu.yaml

```diff
@@ -0,0 +1,2 @@
+---
+openondemand::apache_user: www-data
```

Comment:
Please rename this file to `Debian.yaml` to do a little future proofing.  OnDemand 3.1 will support Debian and Ubuntu so this will be one less thing we have to update for the 3.1 release.